### PR TITLE
Add globe demo

### PR DESCRIPTION
**Pull request by wipfli**

### CODE REVIEW

**Comment by louwers** [comment]
Maybe a control to toggle?

File: globe/index.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta name="viewport" content="initial-scale=1,maximum-scale=1,user-scalable=no" />
+  <script src='https://unpkg.com/maplibre-gl@5.0.0-pre.1/dist/maplibre-gl.js'></script>
```
Comment (HarelM) [inline globe/index.html:5]: I would recommend waiting for an official release...

File: globe/index.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  <style>
+    #map {position: absolute; top: 0; right: 0; bottom: 0; left: 0;}
+  </style>
+</head>
+<body>
+  <div id="map"></div>
+  <script>
+    var map = new maplibregl.Map({
+      container: 'map',
+      style: 'style.json',
```
Comment (HarelM) [inline globe/index.html:16]: I would suggest using an existing style and only setting the protection...

File: globe/index.html (hunk 1 above)
Comment (wipfli) [reply globe/index.html:5]: I think we should have some maplibre-hosted demo for the newsletter. Would you prefer to have it e.g. in the maplibre website instead @HarelM ?

File: globe/index.html (hunk 2 above)
Comment (wipfli) [reply globe/index.html:16]: Is the recommended way to do this something like

```
var map = new ...

map.setProjection("globe");
```

?

Comment (wipfli) [reply globe/index.html:16]: Or would you rather patch the style?

Comment (wipfli) [reply globe/index.html:16]: Like Jakub does it in his demo with something like

```
fetch("https://demotiles.maplibre.org/style.json")
      .then(response => response.json())
      .then(fetchedStyle => {
				fetchedStyle["projection"] = { "type": "globe" };
				map = new maplibregl.Map({
					container: 'map',
					style: fetchedStyle,
					center: [0.0, 0.0],
					zoom: 3.0
				});
      });
```

Comment (HarelM) [reply globe/index.html:16]: I think an API is missing here to solve this, that's why we should wait for an official release.

File: globe/index.html (hunk 1 above)
Comment (HarelM) [reply globe/index.html:5]: The demo can wait for an official release to be made, no need to rush it.
It's only the first pre-release.

File: globe/index.html (hunk 2 above)
Comment (wipfli) [reply globe/index.html:16]: Good point!

**Pull request closed without merging**